src/main/java/frc/trigon/robot/subsystems/leds/LedCommand.java

```diff
@@ -0,0 +1,21 @@
+package frc.trigon.robot.subsystems.leds;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public abstract class LedCommand extends CommandBase {
+    private final LedStrip ledStrip;
+
+    /**
+     * @param ledStrip the led strip
+     */
+    protected LedCommand(LedStrip ledStrip) {
+        this.ledStrip = ledStrip;
+
+        addRequirements(ledStrip);
+    }
+
+    @Override
+    public void initialize() {
+        ledStrip.cancelOverlapping();
+    }
+}
```

Comment:
it looks like you can add:

    @Override
    public void end(boolean interrupted) {
        Color[] colors = new Color[ledStrip.getLength()];
        for (int i = 0; i < ledStrip.getLength(); i++)
            colors[i] = Color.kBlack;
        ledStrip.setLedsColors(colors);
    }

AND

    @Override
    public boolean runsWhenDisabled() {
        return true;
    }

to this class, and remove all these duplicated code from the other concrete LedCommands.